OpenDreamRuntime/DreamMapManager.cs

```diff
@@ -73,7 +73,10 @@ public struct Cell {
 
         public void LoadMaps(List<DreamMapJson> maps) {
             if (maps.Count == 0) throw new ArgumentException("No maps were given");
-            else if (maps.Count > 1) throw new NotImplementedException("Loading more than one map is not implemented");
+            else if (maps.Count > 1)
+            {
+                Logger.Error("Loading more than one map is not implemented, skipping additional maps");
```

Comment:
Seems more like a warning to me
```suggestion
                Logger.Warning("Loading more than one map is not implemented, skipping additional maps");
```